specification/connector/sync_job_update_stats/SyncJobUpdateStatsRequest.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Dictionary } from '@spec_utils/Dictionary'
+import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
+import { RequestBase } from '@_types/Base'
+import { Id } from '@_types/common'
+import { integer } from '@_types/Numeric'
+
+/**
+ * Set the connector sync job stats.
+ * Stats include: `deleted_document_count`, `indexed_document_count`, `indexed_document_volume`, and `total_document_count`.
+ * You can also update `last_seen`.
+ * This API is mainly used by the connector service for updating sync job information.
+ *
+ * To sync data using self-managed connectors, you need to deploy the Elastic connector service on your own infrastructure.
+ * This service runs automatically on Elastic Cloud for Elastic managed connectors.
+ * @rest_spec_name connector.sync_job_update_stats
+ * @availability stack stability=experimental visibility=public
+ * @doc_id connector-sync-job-stats
+ */
+export interface Request extends RequestBase {
+  /**
+   * The sync job to be created
+   */
+  path_parts: {
+    /**
+     * The unique identifier of the connector sync job.
+     */
+    connector_sync_job_id: Id
+  }
+  body: {
+    /**
+     * The number of documents the sync job deleted.
+     */
+    deleted_document_count: integer
+    /**
+     * The number of documents the sync job indexed.
+     */
+    indexed_document_count: integer
+    /**
+     * The total size of the data (in MiB) the sync job indexed.
+     */
+    indexed_document_volume: integer
+    /**
+     * The timestamp to use in the `last_seen` property for the connector sync job.
+     */
+    last_seen?: integer
```

Comment:
```suggestion
    last_seen?: EpochTime<UnitNanos>
```
last_seen seems to be a timestamp with nanosecond precision -  [server code](https://github.com/elastic/elasticsearch/blob/4efeca83b4a9f40f5a8fac9a8a3b4f2191d5282e/x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/action/UpdateConnectorSyncJobIngestionStatsAction.java#L205)